harbor_exporter.go

```diff
@@ -208,6 +214,16 @@ func NewExporter(opts harborOpts, logger log.Logger) (*Exporter, error) {
 		"Get public repositories which are accessed most.).",
 		[]string{"repo_name", "repo_id"}, nil,
 	)
+	replicationStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, opts.instance, "replication_status"),
+		"Get status of the last execution of this replication policy: Succeed = 1, any other status = 0.",
+		[]string{"repl_pol_name"}, nil,
+	)
+	replicationTasks = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, opts.instance, "replication_tasks"),
```

Comment:
can you change the metric name to `replication_tasks_total`?
https://prometheus.io/docs/practices/naming/

Reply:
Thanks for the link, I didn't know about these conventions.
1) It says "Note that an accumulating count has total as a suffix". However, this is not an accumulating count, just the number of tasks in the latest execution of the replication. The unit basically is 'task' (not one of the base units) and the conventions use plural.
2) Also I see "either the sum() or the avg() over all dimensions of a given metric should be meaningful". So it is not logical to have all the separate types _and_ the total as labels.
3) Given the examples under "labels", the label 'type' can better be named 'result' (result of the task: failed, succeed, etc).

So I suggest to remove the label value 'total'; change the label name to 'result'; and keep the name 'replication_tasks'.
Please let me know what you think.

Reply:
the replication metrics are not  accumulating count? I don't have replication on my harbor so I cannot test them.
If it's not a count, I agree to use `replication_tasks` for the metric name

Reply:
done